demisto_sdk/commands/create_id_set/tests/create_id_set_test.py

```diff
@@ -198,42 +203,114 @@ def test_create_command_to_implemented_integration_map(repo):
+    @staticmethod
+    def test_do_not_modify_specific_brand(repo):
+        """
+        Given:
+            * playbook with a command not using a specific brand
+            * playbook with a command using a specific brand
+        When:
+            updating the commands_to_integrations fields in playbooks
+        Then:
+            only update commands that don't have a specific brand
+            only update non generic-commands
+        """
+        integrations = [
+            {
+                'MainInteg': OrderedDict([
+                    ('name', 'MainInteg'),
+                    ('commands', ['generic-command']),
+                ])
+            },
+            {
+                'SecondaryInteg': OrderedDict([
+                    ('name', 'SecondaryInteg'),
+                    ('commands', ['generic-command', 'specific-command']),
+                ])
+            },
+            {
+                'Syslog': OrderedDict([
+                    ('name', 'Syslog'),
+                    ('commands', ['send-notification']),
+                ])
+            },
+            {
+                'Slack': OrderedDict([
+                    ('name', 'Slack'),
+                    ('commands', ['send-notification']),
+                ])
+            },
+        ]
+        playbooks = [
+            {
+                'Playbook1': OrderedDict([
+                    ('name', 'Playbook1'),
+                    ('command_to_integration', {
+                        'specific-command': "",
+                        'generic-command': "",
+                        'send-notification': ""
+                    }),
+                ]),
+            },
+            {
+                'Playbook2': OrderedDict([
+                    ('name', 'Playbook2'),
+                    ('command_to_integration', {
+                        'generic-command': 'MainInteg',
+                        'no-integration': '',
+                        'send-notification': "Slack"
+                    }),
+                ]),
+            },
+        ]
+
+        id_set_creator = IDSetCreator(print_logs=False)
+        id_set_creator.id_set["integrations"] = integrations
+        id_set_creator.id_set["playbooks"] = playbooks
+
+        id_set_creator.add_command_to_implementing_integrations_mapping()
+
+        playbook1 = id_set_creator.id_set['playbooks'][0]['Playbook1']
+        playbook2 = id_set_creator.id_set['playbooks'][1]['Playbook2']
+        assert playbook1['command_to_integration']['specific-command'] == ['SecondaryInteg']
+        assert playbook1['command_to_integration']['generic-command'] == ['MainInteg', 'SecondaryInteg']
+        assert playbook1['command_to_integration']['send-notification'] == ''
+        assert playbook2['command_to_integration']['no-integration'] == ''
+        assert playbook2['command_to_integration']['send-notification'] == 'Slack'
```

Comment:
Not sure I understand this test, it contains more than one use case.
What is playbook 1? what is 2? what are we expecting? 
I suggest splitting it into two tests.